src/main/java/core/basesyntax/dto/FruitRecordDtoParserImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.dto;
+
+import core.basesyntax.service.OperationType;
+import core.basesyntax.service.OperatorParser;
+import core.basesyntax.service.implementation.OperatorParserImpl;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitRecordDtoParserImpl implements FruitRecordDtoParser {
+    private static final String DELIMITER = ",";
+    private static final int INDEX_OF_OPERATION = 0;
+    private static final int INDEX_OF_FRUIT = 1;
+    private static final int INDEX_OF_QUANTITY = 2;
+    private final List<FruitRecordDto> fruitRecordDtos = new ArrayList<>();
+
+    @Override
+    public List<FruitRecordDto> parse(List<String> lines) {
+        fruitRecordDtos.clear();
+        lines.remove(0);
```

Comment:
it's safer to remove by value here, pass title string

Reply:
done